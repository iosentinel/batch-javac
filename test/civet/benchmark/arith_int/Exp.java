package benchmark.arith_int;

public class Exp {
  int eval(int[] env) {
    return this.eval(env);
  }
}
