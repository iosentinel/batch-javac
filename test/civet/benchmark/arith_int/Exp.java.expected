package benchmark.arith_int;

public class Exp {
    
    public Exp() {
        super();
    }
    
    int eval(int[] env) {
        return this.eval(env);
    }
}